internal/test/examples_test.go

```diff
@@ -101,18 +102,3 @@ func TestExample07(t *testing.T) {
 
 	t.Logf("profile:\n%v", p.Profile().String())
 }
-
-func TestExample08(t *testing.T) {
```

Comment:
Removed because of lack of support now?

Reply:
removed because of the new file system, and TestExample08 relied on an existing database file. I didn't think it was worth porting it, because there will be enough tests that test the new file system.
I added it back then just to see a database file really 😄

Reply:
I.e. the same functionality should be tested by 06 and/or 07 I think, 08 just relies on everything being in a file